MushroomObserver/mushroom-observer
# Add nomenclature identifier field to Names

Adds a field for the identifier -- the registration number in fungal nomenclature databases.

- Adds an ICN Identifier field to the Name create/edit form
- Displays the results of that field on the Name page
- If the Name lacks an identifier, the Name page 
  - notes that it's missing and includes links to Index Fungorum and MycoBank searches, or
  - if the Name is clearly unregistrable (e.g., a plant) does not display anything related to the ICN number.

Delivers [PIvotal Story #167748352](https://www.pivotaltracker.com/story/show/167748352)

Suggested manual test:
- View a fungal name that does not have an identifier.
Result: Nomenclature section should display an Identifier missing message, links to IF and MB searches
- Edit that Name, adding the correct Identifier
Result: Nomenclature section should display the identifier, links to the IF and MB records. (MB actually links to the results of a search for the identifier.)
